### PR TITLE
Implemented createWorld

### PR DESCRIPTION
**Pull request by PeachesMLG**
# Description
Implemented Create World
Replacement of https://github.com/MockBukkit/MockBukkit/pull/280

# Checklist
The following items should be checked before the pull request can be merged.
- [ ] Version number updated in `settings.gradle`.
- [ ] Unit tests added.
- [ ] Code follows existing code format.

# Info on creating a pull request
Before the pull request can be accepted, the version number stored in settings.gradle should be updated.
MockBukkit uses semantic-versioning (https://semver.org/)

 - If the PR fixes a bug in MockBukkit, update the patch version. (if the current version is `0.4.1`, the new version should be `0.4.2`)
 - If the PR adds a new feature to MockBukkit, update minor version. (if the current version is `0.4.1`, the new version should be `0.5.0`)
 - Make sure that unit tests are added which test the relevant changes.
 - Make sure that the changes follow the existing code format.

Note that a PR that fixes an `UnimplementedOperationException` should be considered a new version and not a bugfix.

The version number can be found in `settings.gradle`.

### CODE REVIEW

File: src/main/java/be/seeseemelk/mockbukkit/WorldMock.java [hunk 1]
```diff
@@ -14,26 +14,7 @@
...
-import org.bukkit.Particle;
-import org.bukkit.Raid;
-import org.bukkit.Sound;
-import org.bukkit.SoundCategory;
-import org.bukkit.StructureType;
-import org.bukkit.TreeType;
-import org.bukkit.World;
-import org.bukkit.WorldBorder;
-import org.bukkit.WorldType;
+import org.bukkit.*;
```
Comment (TheBusyBiscuit) [inline src/main/java/be/seeseemelk/mockbukkit/WorldMock.java:17]: Please avoid wildcard imports

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]
Aside form the wildcard import, missing unit tests and the fact that you butchered the formatting, it looks sufficient in my opinion! Thanks :)

File: src/main/java/be/seeseemelk/mockbukkit/WorldMock.java (hunk 1 above)
Comment (PeachesMLG) [reply src/main/java/be/seeseemelk/mockbukkit/WorldMock.java:17]: My Ide does all of those automatically, usually there is a script that changes the formatting back but idk why it didnt run this time :shrug:

**Comment by PeachesMLG** [comment]
ffs, well I added Unit Tests and then spent like an hour trying to make my ide play nice with your shitty coding conventions but its not working. Fix them yourself

**Pull request closed without merging**